apps/crn-frontend/src/network/teams/TeamList.tsx

```diff
@@ -27,6 +28,12 @@ const NetworkTeamList: React.FC<NetworkTeamListProps> = ({
     searchQuery,
     filters,
   });
+  usePrefetchWorkingGroups({
+    currentPage: 0,
+    pageSize,
+    searchQuery,
+    filters,
```

Comment:
I've just asked @kmaid about this, we should not preserve the filters.

About my past comment, I was making a confusion with `searchQuery` and `filters` which are not the same.

Since the filters change from one entity to another, we shouldn't preserve then, so actually we should have `filters: new Set()`.

Sorry about the confusion 😅